test/unit/test_task_run_diagnostic_reports.py

```diff
@@ -48,6 +48,50 @@ def test_run_reports(logging, emailer_send):
     emailer_send.assert_has_calls([send_email_call_r1, send_email_call_r2])
 
 
+@patch("src.utils.email.Emailer.send")
+@patch("src.task_run_diagnostic_reports.logging")
+def test_run_reports_with_additional_recipients(logging, emailer_send):
+    run_successfully = ["r1", "r2"]
+    report_responses = {
+        "r1-key": [ReportStatusResult({"status": "success",
+                                       "version": "r1-version",
+                                       "output": None})],
+        "r2-key": [ReportStatusResult({"status": "success",
+                                       "version": "r2-version",
+                                       "output": None})]
+    }
+    ow = MockOrderlyWebAPI(run_successfully, report_responses)
+    wrapper = OrderlyWebClientWrapper(None, lambda x: ow)
+    versions = run_reports(wrapper, MockConfig(), reports,
+                           ["another@recipient.com"])
```

Comment:
Do you think it might be better to use `*additional_recipients` so that you don't have to wrap them in a list? I think it's quite nice for variable length optional arguments at the end like this and have a hunch that it might make interoperability easier when triggering the task from java or the command line

Reply:
Good idea